src/utilities/isPalindrome/isPalindrome.js

```diff
@@ -0,0 +1,6 @@
+export function isPalindrome(text) {
+  return (
```

Comment:
This is extremely difficult to read. Declare additional variables to clean it up.